Documentation/guides/building-apps/build-properties.md

```diff
@@ -1160,6 +1174,36 @@ string.
 
 Added in Xamarin.Android 7.2.
 
+## ApplicationId
+
+An MSBuild property that can be used as an alternative to
+`/manifest/@package` in the `AndroidManifest.xml`
+file. To opt into this feature you must also enable
+`<GenerateApplicationManifest>true</GenerateApplicationManifest>`.
+This will be the default going forward in .NET 6.
```

Comment:
Should we include a note that states this should NOT point to an android resource value e.g @strings/packageName. 
While aapt2 it seems lets you do this, the app will fail to run when installed.

Reply:
I added a comment to all of these like:

> See the [Android documentation][manifest-element] for further details
about the requirements for `/manifest/@package`.

[manifest-element]: https://developer.android.com/guide/topics/manifest/manifest-element

Google has the full list of rules.